dc-cudami-editor/src/components/WebpageForm.jsx

```diff
@@ -81,9 +82,26 @@ const WebpageForm = (props) => {
                     })
                   }}
                 />
+                <Row>
+                  <Col sm="12">
+                    <hr />
+                  </Col>
+                </Row>
               </LanguageTabContent>
             ))}
           </TabContent>
+          <Row>
+            <Col sm="12">
+              <hr />
+            </Col>
+          </Row>
+          <PublicationDatesForm
```

Comment:
Wäre es nicht schöner, wenn die Datumsfelder direkt unterhalb der ID wären?

Reply:
So in etwa:

![image](https://user-images.githubusercontent.com/20651387/83004475-03174900-a010-11ea-8fba-db09170052f5.png)

Reply:
Genau :+1:

Reply:
Mit 8a4e105 ist Kalender nun direkt unter dem ID Feld :)